Source/WebKit/GPUProcess/GPUProcess.cpp

```diff
@@ -548,20 +593,24 @@ void GPUProcess::webProcessConnectionCountForTesting(CompletionHandler<void(uint
 #if PLATFORM(COCOA) && ENABLE(MEDIA_STREAM)
 void GPUProcess::processIsStartingToCaptureAudio(GPUConnectionToWebProcess& process)
 {
-    for (auto& connection : m_webProcessConnections.values())
-        connection->processIsStartingToCaptureAudio(process);
+    for (auto& connection : m_webProcessConnections.values()) {
```

Comment:
I think the variable should be named `connections` since its type is `WebProcessConnections`.